stdlib/math.gr

```diff
@@ -0,0 +1,71 @@
+/* grainc-flags --compilation-mode=runtime */
+import WasmI32 from "runtime/unsafe/wasmi32"
+import WasmI64 from "runtime/unsafe/wasmi64"
+import WasmF64 from "runtime/unsafe/wasmf64"
+import {
+  (+),
+  (-),
+  (/),
+  (*),
+  (>),
+  coerceFloat64,
+  reducedInteger
+} from "runtime/numbers"
+import { newFloat64 } from "runtime/dataStructures"
+
+primitive (||) : (Bool, Bool) -> Bool = "@or"
+primitive throw : Exception -> a = "@throw"
+
+exception UnknownNumberTag
+
+// add
```

Comment:
Could you change these docstrings to match this format? https://github.com/grain-lang/grain/blob/main/stdlib/bytes.gr#L52